vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -193,12 +193,12 @@ type internal FSharpSettingsFactory
                     Width = 360,
                     Height = 120,
                     Window="34E76E81-EE4A-11D0-AE2E-00A0C90FFFC3")>]
-[<ProvideLanguageEditorOptionPage(typeof<OptionsUI.IntelliSenseOptionPage>, "F#", null, "IntelliSense", "6008")>]
-[<ProvideLanguageEditorOptionPage(typeof<OptionsUI.QuickInfoOptionPage>, "F#", null, "QuickInfo", "6009")>]
-[<ProvideLanguageEditorOptionPage(typeof<OptionsUI.CodeFixesOptionPage>, "F#", null, "Code Fixes", "6010")>]
-[<ProvideLanguageEditorOptionPage(typeof<OptionsUI.LanguageServicePerformanceOptionPage>, "F#", null, "Performance", "6011")>]
-[<ProvideLanguageEditorOptionPage(typeof<OptionsUI.AdvancedSettingsOptionPage>, "F#", null, "Advanced", "6012")>]
-[<ProvideLanguageEditorOptionPage(typeof<OptionsUI.FormattingOptionPage>, "F#", null, "Formatting", "6014")>]
+[<ProvideLanguageEditorOptionPage(typeof<OptionsUI.IntelliSenseOptionPage>, "F#", null, "IntelliSense", "6008", "IntelliSensePageKeywords")>]
```

Comment:
For my understanding - what does this change here do? Was it omitted by mistake before?

Reply:
AFAIU this is just the way to connect the keywords (xml data) with the VS settings page UI.

Technically this was not a bug, this was just not making use of the opportunities. Now we use another overload of [the attribute](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.providelanguageeditoroptionpageattribute?view=visualstudiosdk-2022) that allows having some keywords for searching these settings:
![image](https://user-images.githubusercontent.com/5451366/203339583-12c2a77a-e572-4d04-aecf-c814cea71a32.png)